docs/contributors.rst

```diff
@@ -0,0 +1,118 @@
+.. include:: links.rst
+
+------------------------
+Contributing to FMRIPREP
+------------------------
+
+This document explains how to prepare a new development environment and
+update an existing environment, as necessary.
+
+Development in Docker is encouraged, for the sake of consistency and
+portability.
+By default, work should be built off of `poldracklab/fmriprep:latest
+<https://hub.docker.com/r/poldracklab/fmriprep/>`_ (see the
+installation_ guide for the basic procedure for running).
```

Comment:
Sorry, now I realize that I was looking at the github advanced diff... should be fine.